frontend/src/components/layout/Header/Header.test.tsx

```diff
@@ -54,29 +54,35 @@ describe('Header tests', () => {
     process.env = OLD_ENV;
   });
 
-  it('Header renders correctly', () => {
+  it('Header renders correctly', async () => {
```

Comment:
These tests were periodically leaking into the next test due to async actions running in the background.